tests/test_grpc.py

```diff
@@ -148,17 +148,17 @@ def test_heartbeat_denied(participant_stub, coordinator_service):  # pylint: dis
 
 
 @mock.patch("xain_fl.coordinator.heartbeat.threading.Event.is_set", side_effect=[False, True])
-@mock.patch("xain_fl.coordinator.heartbeat.time.sleep", return_value=None)
+@mock.patch("xain_fl.coordinator.heartbeat.threading.Event.wait", return_value=None)
 @mock.patch("xain_fl.coordinator.heartbeat.Coordinator.remove_participant")
-def test_monitor_heartbeats(mock_participants_remove, _mock_sleep, _mock_event):
-    """[summary]
-
-    .. todo:: Advance docstrings (https://xainag.atlassian.net/browse/XP-425)
+def test_monitor_heartbeats(mock_participants_remove, _mock_event_wait, _mock_event_is_set):
+    """Test that when there is a participant with an expired heartbeat,
+    ``Coordinator.remove_participant`` is called exactly once.
 
     Args:
-        mock_participants_remove ([type]): [description]
-        _mock_sleep ([type]): [description]
-        _mock_event ([type]): [description]
+        mock_participants_remove: mock of ``Coordinator.remove_participant()``
```

Comment:
just want to double-check as this one isn't prefixed with underscore, was this a deliberate choice?

Reply:
I just kept the original name, which was without the underscore. I think it makes sense, because this mock is pretty important and we perform checks on it in this test.

Reply:
agreed, ok!